test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -82,7 +82,7 @@ public void testFailure(Failure failure) throws Exception {
         gradleMessageBuilder.appendAllOpts(failure.getDescription());
 
         //Rest tests are a special case as they allow for additional parameters
-        if (ESRestTestCase.class.isAssignableFrom(failure.getDescription().getTestClass())) {
+        if (ESClientYamlSuiteTestCase.class.isAssignableFrom(failure.getDescription().getTestClass())) {
```

Comment:
It never has before - the assumption has been that the cluster information changes when you restart the test. These are things that select the suite to run.